include/MainWindow.h

```diff
@@ -168,6 +169,7 @@ public slots:
 	void toggleFxMixerWin();
 	void togglePianoRollWin();
 	void toggleControllerRack();
+    void updateCountDownDuration();
```

Comment:
Yeah, it's annoying my editor will convert all the tabs to spaces, I will try to configure it not to do so.